client/src/pages/AddEmployee.js

```diff
@@ -0,0 +1,145 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import swal from "sweetalert";
+
+function AddEmployee() {
+  const [form, setForm] = useState({});
+
+  const handleForm = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    axios
+      .post("http://localhost:8000/api/v1/employees/", form)
+      .then((response) => {
+        console.log(response);
+        if (response.status === 201) {
+          swal({
+            title: "Added",
+            text: "Employee Added Successfully",
+            icon: "success",
+            button: "OK!",
+          });
+          setForm({});
+        } else {
+          swal({
+            title: "Failed",
+            text: "Employee Could not be added",
+            icon: "error",
+            button: "OK",
+          });
+        }
+      });
+    document.getElementById("formData").reset();
+  };
+
+  return (
+    <div>
+      <section
+        className="h-100 h-custom"
+        style={{ backgroundColor: "#8fc4b7" }}
+      >
+        <div className="container py-5 h-100">
+          <Link to="/admin">
+            <button
+              className="btn btn-success btn-lg mb-1 position-absolute top-0 end-0"
+              style={{ marginTop: "20px", marginRight: "20px" }}
+            >
+              Back to Dashboard
+            </button>
+          </Link>
+          <div className="row d-flex justify-content-center align-items-center h-100">
+            <div className="col-lg-8 col-xl-6">
+              <div className="card rounded-3">
+                <div className="card-body p-4 p-md-5">
+                  <h3 className="mb-4 pb-2 pb-md-0 mb-md-5 px-md-2">
+                    Add a new Employee
```

Comment:
add a const function for each div, and use that component here, for more readability and cleanliness  of code.